speculation-rules.bs

```diff
@@ -23,6 +23,9 @@ spec: html; urlPrefix: https://html.spec.whatwg.org/multipage/
   type: dfn
     urlPrefix: webappapis.html
       text: script; url: concept-script
+spec: nav-speculation; urlPrefix: prefetch.html
```

Comment:
Hmm, probably this indicates the linking database needs updating to have two separate specs (speculation-rules and prefetch) instead of one (nav-speculation). Might be worth filing an issue on tabatkins/bikeshed for that.

Reply:
I'm not sure I follow. It's at least as likely that I don't know what I'm doing when I want to link between two .bs files in one repository. This seemed to work, but I may be holding it wrong.

Reply:
So in general you should only need to add things to the `<pre class="anchors">` block if they're not in the linking database, either because they're not exported, or because they're not in the linking database.

I was hypothesizing that the linking database only contains https://wicg.github.io/nav-speculation/, and might not contain https://wicg.github.io/nav-speculation/prefetch.html .

However perhaps the issue is just that the linking database doesn't get updated until after the PR is merged. So this is definitely necessary for now. The question is whether, ~24 hours after merging, you can remove it and just use `[=prefetch=]` with nothing in `<pre class=anchors>`.